src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -311,6 +311,28 @@ public void ClearBackground()
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Sets render effect to the view. The effect is applied to at most one view.
+        /// </summary>
+        /// <param name="effect">A render effect to set.</param>
+        /// <since_tizen> 12 </since_tizen>
+        public void SetRenderEffect(RenderEffect effect)
+        {
+            Interop.View.SetRenderEffect(SwigCPtr, RenderEffect.getCPtr(effect));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Clears render effect.
+        /// </summary>
+        /// <since_tizen> 12 </since_tizen>
```

Comment:
```suggestion
         [EditorBrowsable(EditorBrowsableState.Never)]
```

Please make it as hidden